tests/EndToEnd/restrict-content/post-types/RestrictContentProductPageCest.php

```diff
@@ -262,6 +262,13 @@ public function testRestrictContentByInvalidProduct(EndToEndTester $I)
 	 */
 	public function testRestrictContentWhenEditingWithFrontendPageBuilder(EndToEndTester $I)
 	{
+		// Mark test as skipped.
+		$I->clearRestrictContentCookie($I);
+		$I->deactivateThirdPartyPlugin($I, 'convertkit-actions-and-filters-tests');
+		$I->deactivateKitPlugin($I);
+		$I->resetKitPlugin($I);
+		$I->markTestSkipped('Beaver Builder throws a notice regarding translations due to incompatibility with WordPress 6.8. This test cannot reliably be run.');
```

Comment:
I'm curious why we have to clear restricted content, deactivate, etc. before skipping this test?

Reply:
The next test would fail as whilst the `_before` method is always run if you skip a test, the `_passed` method is also skipped, resulting in Plugins remaining activated before the next test (which expects them to be deactivated). Hopefully we can reinstate this test in RC3 or on the 6.8 release.

Reply:
That makes sense - is there any way to directly call the _passed method so your intention is clearer?